---
float64 block_size_side
float64 block_size_height
float64 block_convex_radius
float64 block_convex_height
